diff -up netlabel_tools-0.19/Makefile.new-hdrs netlabel_tools-0.19/Makefile
--- netlabel_tools-0.19/Makefile.new-hdrs	2009-01-05 22:24:24.000000000 +0100
+++ netlabel_tools-0.19/Makefile	2010-06-16 13:36:55.170646610 +0200
@@ -41,8 +41,9 @@ INSTALL_SBIN_DIR = $(INSTALL_PREFIX)/sbi
 INSTALL_BIN_DIR = $(INSTALL_PREFIX)/bin
 INSTALL_MAN_DIR = $(INSTALL_PREFIX)/share/man
 
-OWNER = root
-GROUP = root
+# Mock doesn't allow this.
+# OWNER = root
+# GROUP = root
 
 #
 # targets
@@ -70,15 +71,15 @@ install: $(SUBDIRS)
 	@echo "INFO: installing files in $(INSTALL_PREFIX)"
 	@mkdir -p $(INSTALL_SBIN_DIR)
 	@mkdir -p $(INSTALL_MAN_DIR)/man8
-	@install -o $(OWNER) -g $(GROUP) -m 755 netlabelctl/netlabelctl \
+	@install -m 755 netlabelctl/netlabelctl \
 	 $(INSTALL_SBIN_DIR)/netlabelctl
-	@install -o $(OWNER) -g $(GROUP) -m 644 docs/man/netlabelctl.8 \
+	@install -m 644 docs/man/netlabelctl.8 \
 	 $(INSTALL_MAN_DIR)/man8
 
 $(VERSION_HDR): version_info
 	@echo "INFO: creating the version header file"
 	@hdr="$(VERSION_HDR)"; \
-	. version_info; \
+	. ./version_info; \
 	echo "/* automatically generated - do not edit */" > $$hdr; \
 	echo "#ifndef _VERSION_H" >> $$hdr; \
 	echo "#define _VERSION_H" >> $$hdr; \
